### DISCUSSION TITLE
Removing all the virtual trackers from SteamVR?

### DISCUSSION THREAD

**Discussion by HexHyte** [KinectToVR/Amethyst-Releases#3] [Q&A]
The title is self explanatory, so basically i finally bought the actual 3.0 trackers and i want to remove all the virtual trackers created by Amethyst from SteamVR... but ehm, I still haven't figured out how to do that.
Any help?

**Comment by hyblocker** [comment]
Just disable the Amethyst add-on in SteamVR Settings => Startup => Add-ons

**Reply by AuroraNemoia** [reply]
Sounds like they might be talking about the manage trackers window on SteamVR.

You can remove those in `C:\Program Files (x86)\Steam\config\steamvr.vrsettings` in the `"trackers"` section. It's just a JSON file

You also need to disable the Amethyst driver first as Hekky said. The driver when it loads will try registering all the trackers supported by Amethyst again.
